lib/new-admin/services/transactions.js

```diff
@@ -24,23 +24,44 @@ function addNames (txs) {
 
 const camelize = _.mapKeys(_.camelCase)
 
-function batch (from = new Date(0).toISOString(), until = new Date().toISOString(), limit = null, offset = 0, id = null) {
+function batch (
+  from = new Date(0).toISOString(),
+  until = new Date().toISOString(),
+  limit = null,
+  offset = 0,
+  id = null,
+  txClass = null,
+  machineName = null,
+  customerName = null,
+  fiatCode = null,
+  cryptoCode = null,
+  toAddress = null,
+  status = null
+) {
   const packager = _.flow(_.flatten, _.orderBy(_.property('created'), ['desc']), _.map(camelize), addNames)
 
   const cashInSql = `select 'cashIn' as tx_class, txs.*,
```

Comment:
I know this wasn't you but since we're already messing around with this query it's a good chance of making stuff uppercase to be more consistent with the rest of the codebase